.github/dependabot.yml

```
@@ -0,0 +1,8 @@
+version: 2
+
+updates:
+  - package-ecosystem: "pip"
+    directory: "/"
+    open-pull-requests-limit: 5
+    schedule:
+      interval: "daily"
```

```suggestion
      interval: "weekly"
```

What's your advice on the frequency of this? As a maintainer, I can rarely react in 1 day anyways, so I just find it fitting to do this weekly?

---

Once a week is fine. Merged dependency bumps are not really worth much, unless you release them :)

---

Yes exactly.. but they also often requires to ponder if they are actually worth a release or just noisy :)